type.md

```diff
@@ -2,8 +2,11 @@
 title: Type System
 ---
 
-Common Lisp has a complete and flexible type system and corresponding tools to
-inspect, check and manipulate types.
+Common Lisp has a complete and flexible type system and corresponding
+tools to inspect, check and manipulate types. It allows to create
```

Comment:
It allows creating. Gerund form, not the infinitive.

Reply:
thanks!